ci-operator/jobs/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main-presubmits.yaml

```diff
@@ -136,6 +136,86 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )rosa-sts-advanced-critical-high-presubmit,?($|\s.*)
+  - agent: kubernetes
```

Comment:
Can you please clarify what is the difference between this to the second job in this PR?

Reply:
One for rosa-sts-pl (private) profile , another - for rosa-sts-ad (advanced)

Reply:
Ok, got it
You can resolve this